flow-typed/platform.js

```diff
@@ -16,6 +16,24 @@ declare type MWPState = {
 		enableServiceWorker: boolean,
 		requestLanguage: string,
 		supportedLangs: Array<string>,
+		member: { 
+			id: number,
+			status: number,
+			timestamp: number,
+			bs: number,
+			// time zone
+			tz: string,
+			zip: number,
+			country: string,
+			city: string,
+			state: string,
```

Comment:
I would guess that `state` is not populated for all members, so is this param optional, i.e. `state?: string`?

Are there other optional keys?